prdoc/pr_6039.prdoc

```diff
@@ -0,0 +1,41 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: "Added Trusted Query API calls."
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      Added is_trusted_reserve and is_trusted_teleporter API calls to all the runtimes.
+
+  - audience: Runtime User
+    description: |
+      Now you can query any runtime to check if pair<Asset, Location> is either a trusted reserve or trusted teleporter.
```

Comment:
```suggestion
      There's a new runtime API to check if a chain trust a Location as a reserve or teleporter for a given Asset.
      It's implemented in all the relays and system parachains in Westend and Rococo.
```

I'd change the wording to make it easier for dapp developers to understand what this API does.